src/ILLink.Tasks/CreateRuntimeRootDescriptorFile.cs

```diff
@@ -288,34 +403,48 @@ void OutputXml (string iLLinkTrimXmlFilePath, string outputFileName)
 			doc.Save (outputFileName);
 		}
 
-		void AddClass (string classNamespace, string className, string classId, bool keepAllFields = false)
+		void AddClass (string classNamespace, string className, string classId, bool keepAllFields = false, bool featureSwitchOn = false, FeatureSwitchMembers featureSwitch = null)
```

Comment:
Why not use `featureSwitch == null` as the indicator for FS presence - I don't see a need for another parameter.